apps/platform/src/components/dashboard/project/editProjectSheet/index.tsx

```diff
@@ -12,19 +13,162 @@ import { Button } from '@/components/ui/button'
 import { Label } from '@/components/ui/label'
 import { Input } from '@/components/ui/input'
 import { Switch } from '@/components/ui/switch'
-import { editProjectOpenAtom } from '@/store'
+import { editProjectOpenAtom, selectedProjectAtom } from '@/store'
+import { ProjectController } from '@keyshade/api-client'
+
+interface FormData {
```

Comment:
This interface is not needed. We have a dedicated schema for this. You can use `UpdateProject` from `@keyshade/schema`